packages/mui-material/src/locale/index.ts

```diff
@@ -814,7 +814,7 @@ export const enUS: Localization = {
         // if (type === 'previous') {
         return 'Go to previous page';
       },
-    },
+    }},
```

Comment:
Could you revert changes to the above lines? I don't know why they've been modified, but let's not introduce any changes beyond what's necessary.

Reply:
This is the template I copied and it was missing a closing curly bracket. If others try to add locales copying this they will also get an error.

Reply:
Fair enough :)